src/fu-engine.c

```diff
@@ -6054,6 +6056,7 @@ fu_engine_get_host_security_events(FuEngine *self, guint limit, GError **error)
 
 	g_return_val_if_fail(FU_IS_ENGINE(self), NULL);
 
+#if JSON_CHECK_VERSION(1, 6, 0)
 	attrs_array = fu_history_get_security_attrs(self->history, limit, error);
```

Comment:
I think you need to ifdef the

	g_autoptr(GPtrArray) attrs_array = NULL;

otherwise you'll get a clang -Wunused warning. Then LGTM, thanks.